node-packages/commons/src/api.ts

```diff
@@ -967,6 +967,31 @@ export async function getEnvironmentByOpenshiftProjectName(
   return result;
 }
 
+export async function getBackupEnvironment(
+  openshiftProjectName: string,
+  baas_bucket_name: string
+): Promise<any[]> {
+  const result = await graphqlapi.query(`
+    {
+      backupEnvironment(openshiftProjectName: "${openshiftProjectName}", baas_bucket_name: "${baas_bucket_name}") {
+        id,
+        name,
+        project {
+          name
+        }
+      }
+    }
+  `);
+
+  if (!result || !result.environmentByOpenshiftProjectName) {
```

Comment:
The `result` is a data structure that matches the fields you queried, so here `result.environmentByOpenshiftProjectName` will always be undefined because the query you are running is now called `backupEnvironment`.